api/attribute.proto

```diff
@@ -841,10 +841,10 @@ message UnknownAttribute {
   bytes value = 3;
 }
 
-// https://tools.ietf.org/html/draft-dawra-bess-srv6-services-02#section-2.1.1
+// https://www.rfc-editor.org/rfc/rfc9252.html#section-3.2.1
 message SRv6StructureSubSubTLV {
-  uint32 local_block_length = 1;
-  uint32 local_node_length = 2;
+  uint32 locator_block_length = 1;
+  uint32 locator_node_length = 2;
   uint32 function_length = 3;
```

Comment:
rename?

Reply:
Thank you for your review. I'll add locator_* as new field and mark old fields as deprecated.

Reply:
This feature is still based on RFC drafts. Breaking is fine by me.

Reply:
If gobgp prefers RFC-compliant to API backward compatibility, I think we can rename these fields. Because this PR is based on RFC9252.